<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPasswordDialog</class>
 <widget class="QDialog" name="SetPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set/Reset Password</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Old Password
(Blank if not set)</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLineEdit" name="lineEdit_oldPassword">
       <property name="maximumSize">
        <size>
         <width>384</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>New Password</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLineEdit" name="lineEdit_newPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Confirm New Password</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignTop">
      <widget class="QLineEdit" name="lineEdit_confirmPassword">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
